### PR TITLE
Added files for github repo

### PR DESCRIPTION
**Pull request by jorik12345**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## 2. Configuration
+
+Copy the file `/webpack/example.webpack.settings.js` to `/webpack/webpack.settings.js`.
+Additionally change `.env.example` to `.env` if you want settings for local development.
+
+Make sure you copy the following files to your project's root folder:
+
+```bash
+cp /node_modules/superbrave/frontend-build-script/.eslintrc ./.eslintrc
```
Comment (kiettran) [inline README.md]: What if you have a different root directory set up? Then this command cannot be executed.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+This repository is intended as an opiniated approach to building assets and configuring linters. The linter settings therefor cannot be changed.
+
+This repo is using Prettier to automatic format code to prevent the obvious style typos.
+
+## 1. Installation
+
+Use the following command to include the build script in your project.
+
+```
+yarn add https://github.com/superbrave/frontend-build-script
```
Comment (kiettran) [inline README.md]: Would be nice if we can just run `yarn add superbrave/frontend-build-script`. Can fix that before you merge this pull request? Have a look at the documentation on [Yarn](https://yarnpkg.com/en/docs/publishing-a-package).

File: README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+Or if you are using NPM:
+
+```
+npm i https://github.com/superbrave/frontend-build-script
```
Comment (kiettran) [inline README.md]: Can you also have a look on the documentation of [npm](https://docs.npmjs.com/packages-and-modules/) for publishing this package?

**Comment by stixx** [comment]
I'm getting vulnerability alerts for lodash: https://github.com/superbrave/frontend-build-script/network/alert/yarn.lock/lodash/open - can you check this please?

**Comment by jorik12345** [comment]
I'm on that. The problem is that some packages have the old Lodash version as a dependency. So I can;t fix this in this PR but it has to be done in the repo of the packages.

**Pull request merged**